specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2019-05-01/NasCluster.json

```diff
@@ -0,0 +1,207 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Nas Cluster operation endpoints and objects.",
+    "title": "FabricAdminClient",
+    "version": "2019-05-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Fabric.Admin/fabricLocations/{location}/nasClusters/{nasCluster}": {
```

Comment:
>fabricLocations/{location} [](start = 105, length = 26)

Why are you using LocationBased routing instead of modeling it as a tracked resource (which supports routing based on the `location` property in the resource body).

Reply:
That is an overall design of AzureStack Hub API. We put the location in url

Reply:
The administrator API has been designed considering the future case of managing multi-region azure stack instances. Many of the resources are location based proxy resources

Reply:
Makes sense. Thanks for explaining.

---
In reply to: [379570914](https://github.com/Azure/azure-rest-api-specs/pull/8339#discussion_r379570914) [](ancestors = 379570914)